odoo/addons/base/ir/ir_mail_server.py

```diff
@@ -119,7 +119,11 @@ def encode_addr(addr):
         # Header as a string, using an unlimited line length.", the old one
         # was "A synonym for Header.encode()." so call encode() directly?
         name = Header(pycompat.to_text(name)).encode()
-        return formataddr((name, email))
+        # if the from does not follow the (name <addr>),* regex, we might try to encode meaningless strings as address
+        try:
+            return formataddr((name, email))
+        except UnicodeEncodeError:
+            return ""
```

Comment:
Log the error

Reply:
Also shouldn't the output be filtered out of the header somehow?